### PR TITLE
report coverate to Coveralls

### PR DESCRIPTION
**Pull request by PetrDlouhy**
This repo must be enabled on Coveralls

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -7,7 +7,8 @@ deps =
     -rrequirements.txt
     pyftpdlib
     pytest-xdist
+    pytest-cov
 commands =
-    py.test {posargs:--tb=short tests}
+    py.test {posargs:--tb=short --cov=linkcheck" tests}
```
Comment (anarcat) [inline tox.ini]: this introduces a syntax error that makes tests fail, i believe.

**Comment by mgedmin** [comment]
:+1: (after the typo is fixed)

**Comment by PetrDlouhy** [comment]
The typo is fixed, but the repo is not enabled on Coveralls, so the report is not visible.

**Comment by mgedmin** [comment]
I think @anarcat is the only one who can enable coveralls.io integration.

(I use coveralls.io on a few of my other projects, but I don't have the permissions to enable it for this one.  I'm not sure what permission exactly is missing, I just know that this organisation doesn't show up for me in the Coveralls UI.)

**Comment by anarcat** [comment]
@mgedmin that's it, you're promoted to owner. ;)

**Comment by mgedmin** [comment]
Coveralls are enabled.  I've restarted the Travis build to re-trigger the coverage data upload. You can see a report at https://coveralls.io/github/linkcheck/linkchecker.

The badge will show "unknown" until this PR is merged to master:
 [Coverage Status](https://coveralls.io/github/linkcheck/linkchecker?branch=master)

@anarcat: only your review is blocking this merge now.

**Comment by mgedmin** [comment]
(I've now turned off those annoying coveralls comments on pull requests.)

**Comment by mgedmin** [comment]
I've asked Coveralls to indicate a build failure if coverage decreases by 1%.

I'm curious: @anarcat, can you access https://coveralls.io/github/linkcheck/linkchecker/settings ?  If you've never used coveralls before, it's useful to know that you can log in using your GitHub account via some OAUTH magic.  I'd also be curious to know if you have to go to your GitHub profile settings and grant the Coveralls application access to the linkcheck organization (I've done this, but I don't know if that's a global thing or something for each GitHub user who wants to log in to Coveralls).

**Comment by anarcat** [comment]
On 2017-02-14 06:42:36, Marius Gedminas wrote:
> I've asked Coveralls to indicate a build failure if coverage decreases by 1%.

fine with me!

> I'm curious: @anarcat, can you access
> https://coveralls.io/github/linkcheck/linkchecker/settings ?

I can, thanks!

> If you've never used coveralls before, it's useful to know that you
>  can log in using your GitHub account via some OAUTH magic.

and done.

>  I'd also be curious to know if you have to go to your GitHub profile settings and grant the Coveralls application access to the linkcheck organization (I've done this, but I don't know if that's a global thing or something for each GitHub user who wants to log in to Coveralls).

i believe it is global. scary. :)

-- 
Nothing incites to money-crimes like great poverty or great wealth.
                        - Mark Twain

**Comment by mgedmin** [comment]
Repeating so this doesn't get lost: @anarcat: only your review is blocking this merge now.  ;)

**Pull request merged**